### ISSUE TITLE
Add ability to not care about time

### ISSUE THREAD

**Issue by corysimmons** [jefferyleo/f.lux#35]
I just want to open Flux with Alfred, and when it's open, my screen is dimmed to a single preset.

Then when I exit out of Flux, I want it to return to normal brightness.

I sleep at completely random times and "When I wake up" is useless. Messing with settings so that I can turn it on/off causes me to simply not use this app as much as I'd like to.

**Comment by corysimmons** [comment]
Ideally just get rid of all the options and just have this UI:

![image](https://user-images.githubusercontent.com/224636/55946345-8f9f5600-5c1a-11e9-8568-f64d8f311cec.png)

In lieu of breaking Flux, would it be possible to do a separate release like "Flux Lite"? It'd be easy to maintain. :D

**Comment by Joshfindit** [comment]
Suggestion to make this work:

- be able to disable “automatic changing”
- allow users to run it with a command-line switch like “flux —temperature 2800K”

That way @corysimmons and other users could simply script via any scripting tool.

**Comment by corysimmons** [comment]
Yeah, tbh exposing a dead simple API would be fine with me. I could wrap in an Electron app and call it Flux Lite or something if you'd like.
